src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OptionalAttribute, WritableAttribute, OptionalWritableAttribute, Attribute, Cluster, OptionalCommand, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvEnum, TlvField, TlvNullable, TlvObject, TlvUInt16, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5.8 */
+export const OptionsBitmap = {
```

Comment:
Why is this exported? Not sure there is a need to reference this bitmap directly from outside this class.

Why this is not the TlvBitmap object? The name suggests it is, but it is not...
export const OptionsBitmap = TlvBitmap(TlvUInt8, {

Reply:
Yes it should be a TlvBitmap